packages/dds/tree/src/test/tsconfig.json

```diff
@@ -10,6 +10,7 @@
 		"module": "Node16",
 		"moduleResolution": "Node16",
 		"types": ["node", "mocha"],
+		"customConditions": ["allow-ff-test-exports"],
```

Comment:
seems like this tsconfig should extend the common test one, but I don't want to touch that as part of this PR.

Reply:
#20788 does this, but since there's a nonzero chance that we'll want to port this PR, I'd prefer to merge this PR first and then have #20788 remove the added line to tree's test tsconfig.